### PR TITLE
Destination read

### PR DESCRIPTION
**Pull request by vTrip**
Loading the destinations

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -75,66 +75,28 @@ angular.module('awayapp')
...
     TripService.add(newTrip).then(function(t) {
       trips.value.push(t);
-      return addModalPromise;
+      return addTripModalPromise;
     }).then(function(m) {
       m.hide();
     });
-  }
 
-  //Cleanup the modal when we're done with it!
```
Comment (jsuresh) [inline www/js/controllers.js]: Why is this deleted?

**Comment by jsuresh** [comment]
What's this PR meant to be doing?

**Comment by vTrip** [comment]
Hey man, please ignore that PR. I haven't split that massive PR like we spoke about because I went through some major weird bug after following the steps you sent me. After it somehow I lost some of the ionic libraries from the last 7 branches created. Will have to show you tomorrow what I mean. But all is working as if nothing happened so all good on that front. 

Regards,

Vince
